core/domain/skill_services.py

```diff
@@ -230,6 +232,167 @@ def get_all_skill_summaries():
     return skill_summaries
 
 
+def get_filtered_skill_dicts(status, classroom_name, keywords, sort_by):
+    """Returns all the skill summary dicts after filtering.
+
+    Args:
+        status: string. The status of the skill.
+        classroom_name: string. The classroom_name of the topic
+            to which skill is assigned to.
+        keywords: list(string). The keywords to look for
+            in the skill description.
+        sort_by: string. A string indicating how to sort the result.
+
+    Returns:
+        list(dict). The list of skill summary dicts matching the filters.
+    """
+    skill_summary_dicts = get_all_skill_summary_dicts_with_topic_and_classroom()
+
+    filtered_skill_summary_dicts = filter_skills_by_status(
+        skill_summary_dicts, status)
+    filtered_skill_summary_dicts = filter_skills_by_classroom(
+        filtered_skill_summary_dicts, classroom_name)
+    filtered_skill_summary_dicts = filter_skills_by_keywords(
+        filtered_skill_summary_dicts, keywords)
+
+    if sort_by == (
+            constants.TOPIC_SKILL_DASHBOARD_SORT_OPTIONS['DecreasingCreatedOn']):  # pylint: disable=line-too-long
```

Comment:
As a general rule, **never** use pylint disable pragmas. Find a way to not do so, e.g. declare new variables with shorter names or break after '['.

Reply:
Done.